earthaccess/results.py

```diff
@@ -159,15 +159,16 @@ def landing_page(self) -> str:
     def get_data(self) -> List[str]:
         """
         Returns:
-            Returns the GET DATA links, usually a link to a landing page, a DAAC portal or an FTP location.
+            The GET DATA links (usually a landing page link, a DAAC portal, or an FTP location).
         """
         links = self._filter_related_links("GET DATA")
         return links
 
     def s3_bucket(self) -> Dict[str, Any]:
         """
         Returns:
-            Returns the S3 bucket information if the collection has it (**cloud hosted collections only**)
+            The S3 bucket information if the collection has it.
+            (**cloud hosted collections only**)
```

Comment:
Needs indentation!

Reply:
Confirmed that indentation is needed for it to display correctly in https://earthaccess--448.org.readthedocs.build/en/448/